CONTRIBUTING.md

```diff
@@ -0,0 +1,229 @@
+Contributing to the Project
+===================================
+
+Thank you for your interest in contributing to the IdentityHub!
```

Comment:
IdentityHub here is intended? It's MVD repository

Reply:
No it is not intended, I fixed it. Thanks!